public/app/features/explore/Wrapper.tsx

```diff
@@ -30,10 +31,12 @@ const styles = {
 
 function Wrapper(props: GrafanaRouteComponentProps<{}, ExploreQueryParams>) {
   useExplorePageTitle();
+  const { correlations } = useSelector((state) => state.explore);
   const dispatch = useDispatch();
   const queryParams = props.queryParams;
-  const { keybindings, chrome } = useGrafana();
+  const { keybindings, chrome, config } = useGrafana();
   const navModel = useNavModel('explore');
+  const { get } = useCorrelations();
```

Comment:
can we alias this to something more specific?

Reply:
Hmm... maybe we should rename it if we always want to alias it?